concent_api/core/message_handlers.py

```diff
@@ -781,7 +792,7 @@ def handle_send_force_payment(
         subtask_results_accepted.payment_ts for subtask_results_accepted in client_message.subtask_results_accepted_list
     )
 
-    if requestors_claim_payment_info.amount_pending == requestors_claim_payment_info.amount_paid == 0:
+    if claim_against_requestor is None:
```

Comment:
if -> elif ?

and in last else maybe assert, that `claim_against_requestor` is `DepositClaim` instance. I know it is not necessary now, but if something will change.. Such a sanity check. If you don't want to do this it is OK for me, just asking what do you think :)

Reply:
I don't know what should be put in `elif` as condition. Actually it should be only `if`, without `else` because if `if` condition is not met, then we should simply continue the flow. But it is not my part of the code, and I think we have somewhere in coding style to use `else` even if it is not necessary.

Reply:
@rwrzesien You probably mean the discussion about `return` and this case in particular: https://github.com/Code-Poets/coding-style/issues/6#issuecomment-396263483.

That's not the case we have here. Your `if`s are not nested.

I don't have a preference here For me either version is fine.